code/modules/antagonists/changeling/powers/become_headslug.dm

```diff
@@ -30,6 +31,11 @@
 /datum/action/changeling/headslug/proc/become_headslug(mob/user)
 	var/datum/mind/M = user.mind
 	var/list/organs = user.get_organs_zone("head", 1)
+	if(istype(user.loc, /obj/machinery/gibber) || istype(user.loc, /obj/structure/crematorium))
+		var/obj/thing_to_break = user.loc
+		user.forceMove(get_turf(user))//Get them outside of it before it breaks, to prevent issues / so they burst out of it dramatically
```

Comment:
```suggestion
		user.forceMove(get_turf(user)) // Get them outside of it before it breaks, to prevent issues / so they burst out of it dramatically
```